dev/tools/bin/generate_gradle_lockfiles.dart

```diff
@@ -214,3 +224,11 @@ include ":app"
 
 apply from: "${settings.ext.flutterSdkPath}/packages/flutter_tools/gradle/app_plugin_loader.gradle"
 ''';
+
+const String wrapperGradleFileContent = r'''
+distributionBase=GRADLE_USER_HOME
+distributionPath=wrapper/dists
+zipStoreBase=GRADLE_USER_HOME
+zipStorePath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-7.6.2-all.zip
```

Comment:
nit I think 7.6.3 is the most recent patch version.

Reply:
Also does this string need copyright information?

Reply:
Changed to 7.6.3. I checked some of our gradle-wrapper.properties files and it seems they currently don't have copyright headers, so I think it should be fine to land this as is